src/services/WebSocketService.js

```diff
@@ -120,7 +117,7 @@ webSocketsService.install = function (Vue) {
   Vue.config.globalProperties.$addWsOnMessageListener = function (
     listenerFunction
   ) {
-    wsOnmessageListeners = listenerFunction; //.push(listenerFunction);
+    wsOnmessageListeners = listenerFunction; 
```

Comment:
Can you make `wsOnmessageListeners` an array and then push `listenerFunction` to it? In this case, you can add many listeners